doc/langref.html.in

```diff
@@ -2248,12 +2254,11 @@ or
       {#code|test_packed_struct_equality.zig#}
 
       <p>
-      Using packed structs with {#link|volatile#} is problematic, and may be a compile error in the future.
-      For details on this subscribe to
-      <a href="https://github.com/ziglang/zig/issues/1761">this issue</a>.
-      TODO update these docs with a recommendation on how to use packed structs with MMIO
-      (the use case for volatile packed structs) once this issue is resolved.
-      Don't worry, there will be a good solution for this use case in zig.
+      Packed structs can be used to interact with memory-mapped input-output (MMIO), which is
+      common in embedded applications. A pointer of the correct alignment and address to a packed struct
```

Comment:
I think having an illustration of MMIO is on balance good, but there's too much detail here. I'd cut this to:

> Packed structs can be used to interact with memory-mapped input-output (MMIO):

Followed by the illustration code.  If you think the detail about the pointer alignment and address is important for understanding the code snippet (perhaps) it could be a comment in the code snippet.